code/game/objects/items/toys.dm

```diff
@@ -1032,20 +1032,39 @@
 	var/cooldown = 0
 
 /obj/item/toy/nuke/attack_self(mob/user)
-	if (cooldown < world.time)
+	if (obj_flags & EMAGGED)
```

Comment:
this check happens before the cooldown check, so you can spam-use an emagged toy nuke

Reply:
I can re-add the check, but because the explosion deletes the toy nuke, this isn't relevant.